verifier/src/main/java/com/aws/jverify/verifier/JavaToDafnyCompiler.java

```
@@ -466,7 +551,7 @@ BinaryExprOpcode toDafny(Symbol.OperatorSymbol operator) {
             case "||" -> BinaryExprOpcode.Or;
             case "&&" -> BinaryExprOpcode.And;
             case "%" -> BinaryExprOpcode.Mod;
-            default -> throw new NotImplementedException("Operator" + operator.name);
+            default -> null;
```

Should we report an error in this case?

---

yes, but It's already reported by the caller since it has the right source location information